source/slang/hlsl.meta.slang

```diff
@@ -375,30 +375,37 @@ double asdouble(uint lowbits, uint highbits);
 
 // Reinterpret bits as a float (HLSL SM 4.0)
 
+// GLSL Scalar
 __target_intrinsic(glsl, "intBitsToFloat")
 float asfloat(int x);
 __target_intrinsic(glsl, "uintBitsToFloat")
 float asfloat(uint x);
+__target_intrinsic(glsl, "$0")
```

Comment:
This is fine, although we should consider mapping all these functions to `kIRPseudoOp_Pos`, which is what we use for unary `operator+`. That pseudo-op turns into a no-op/identity in the IR, which could enable further optimizations (whereas having `asfloat` on a `float` show up as an intrinsic could stop constant folding and other optimizations).